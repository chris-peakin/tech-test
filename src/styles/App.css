.App {
  display: flex;
  flex-direction: column;
}

.nasa-logo{
  width: 50%;
  margin: auto;
}