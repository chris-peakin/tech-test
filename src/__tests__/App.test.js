import renderer from "react-test-renderer";
import App from "../components/App";

describe("App", () => {
  const testRenderer = renderer.create(<App />);
  it("app renders correctly", () => {
    expect(testRenderer).toMatchSnapshot();
  });
});