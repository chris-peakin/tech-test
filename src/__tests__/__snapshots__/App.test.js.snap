// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App app renders correctly 1`] = `
<div
  className="App"
>
  <img
    alt="nasaLogo"
    className="nasa-logo"
    src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
  />
  <div
    className="Search"
  >
    <form
      className="searchForm"
      onSubmit={[Function]}
    >
      <input
        className="search-input"
        onChange={[Function]}
        type="text"
      />
      <button
        className="search-button"
        type="submit"
      >
        Tally Ho!
      </button>
    </form>
  </div>
  <p
    className="paragraph"
  >
    Either a term has not yet been entered, or there are no results for that particular term. Type a search term above and click "Tally Ho!".
  </p>
</div>
`;
